app/views/female_allocations/new.html.erb

```diff
@@ -0,0 +1,81 @@
+<%= link_to "Back", 'javascript:history.back()', class: "govuk-back-link govuk-!-margin-top-0 govuk-!-margin-bottom-6" %>
+
+<% if @prisoner.needs_a_com? %>
+  <%= render partial: 'shared/notifications/offender_needs_a_com', locals: {offender: @prisoner, email_history: @emails_sent_to_ldu} %>
+<% end %>
+
+<h1 class="govuk-heading-l govuk-!-margin-bottom-5">Allocate a POM</h1>
+
+<div class="govuk-inset-text">
+  For this case we recommend that you allocate a <b><%= recommended_pom_type_label(@prisoner) %> POM</b>.<br/>
+  This takes into account <%= @prisoner.first_name %> <%= @prisoner.last_name %>'s current tier and time left to serve.
+</div>
+
+<% if @previous_poms.any? %>
+  <div class="govuk-warning-text">
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+    <strong class="govuk-warning-text__text">
+      <span class="govuk-warning-text__assistive">Warning</span>
+      <% @previous_poms.each_with_index do |pom, index| %>
+        <%= 'and' unless index.zero? %>
+        <strong><%= pom.first_name %> <%= pom.last_name %></strong>
+      <% end %>
+      have previously been allocated to this case.
```

Comment:
Looks like this will read a little clumsily if there are more than 2 previous POMs:

> **John Smith** and **Jane Doe** and **Fred Bloggs** and **Hannah Claire** have previously been allocated to this case.

Not perfect English! Would be nicer to have commas with an 'and' for the final name.

Looks like the Active Support [`Array#to_sentence` method](https://api.rubyonrails.org/v6.1.3.1/classes/Array.html#method-i-to_sentence) should be able to do the hard work for us:

```suggestion
      <strong><%= @previous_poms.map { |p| "#{p.first_name} #{p.last_name}" }.to_sentence %></strong>
      <%= @previous_poms.size > 1 ? 'have' : 'has' %>
      previously been allocated to this case.
```

Will give us:

> **John Smith, Jane Doe, Fred Bloggs, and Hannah Claire** have previously been allocated to this case.